### PR TITLE
Added documentation for the priority access decision strategy

### LINKED ISSUE #12830: Added access decision strategy to respect voter priority
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#34548
| PR author(s) | @aschempp
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by aschempp**
fixes https://github.com/symfony/symfony-docs/issues/12830

### CODE REVIEW

File: components/security/authorization.rst [hunk 1]
```diff
@@ -49,14 +49,17 @@ recognizes several strategies:
 ``unanimous``
     only grant access if none of the voters has denied access;
 
+``priority``
```
Comment (HeahDude) [inline components/security/authorization.rst]: We also need a `versionadded` directive

**Review by HeahDude — CHANGES_REQUESTED** [review]
Thanks for your contributions., looks good!

File: components/security/authorization.rst (hunk 1 above)
Comment (HeahDude) [reply components/security/authorization.rst]: You can see how here https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations

**Comment by aschempp** [comment]
I have added the versionadded directive

File: components/security/authorization.rst [hunk 2]
```diff
@@ -49,14 +49,22 @@ recognizes several strategies:
...
+    grants or denies access by the first voter that does not abstain;
+
+    .. versionadded:: 5.1
+
+    The priority version strategy was introduced in Symfony 5.1.
```
Comment (OskarStark) [inline components/security/authorization.rst]:
```suggestion
        The priority version strategy was introduced in Symfony 5.1.
```

File: components/security/authorization.rst [hunk 3]
```diff
@@ -49,14 +49,22 @@ recognizes several strategies:
...
+
+
```
Comment (OskarStark) [inline components/security/authorization.rst]:
```suggestion

```

File: security/voters.rst [hunk 4]
```diff
@@ -274,7 +274,16 @@ strategies available:
...
-    config option (which defaults to ``false``).
+    config option (which defaults to ``false``);
+
+``priority``
+    This grants or denies access by the first voter that does not abstain,
+    based on their service priority;
+
+    .. versionadded:: 5.1
+
+    The priority version strategy was introduced in Symfony 5.1.
```
Comment (OskarStark) [inline security/voters.rst]:
```suggestion
        The priority version strategy was introduced in Symfony 5.1.
```

**Review by HeahDude — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Pull request merged**